### PR TITLE
Fixed some translations

### PR DESCRIPTION
**Pull request by Blask0**
I fixed some translations that I think you should change.

#### What is the purpose of this pull request?
To fix the wrong translations and adding new ones.
#### What problem is this solving?
Wrong translations,

Want to help the community and get involved into GitHub

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: messages/es.json [hunk 1]
```diff
@@ -167,10 +167,10 @@
   "promotions.promotion.restriction.origin.marketplace": "Marketplace",
   "promotions.promotion.restriction.origin.fulfillment": "Fulfillment",
```
Comment (Blask0) [inline messages/es.json]: Cumplimiento

File: messages/es.json [hunk 2]
```diff
@@ -167,10 +167,10 @@
   "promotions.promotion.restriction.origin.marketplace": "Marketplace",
```
Comment (Blask0) [inline messages/es.json]: Mercado

File: messages/es.json [hunk 3]
```diff
@@ -13,10 +13,10 @@
   "promotions.promotion.generalInfo.status": "Status",
```
Comment (Blask0) [inline messages/es.json]: Maybe you want to put "Estado" here.

**Comment by Blask0** [comment]
All is fixed, sorry for the chaos... I am new in this 👍

File: messages/es.json [hunk 4]
```diff
@@ -101,33 +101,33 @@
...
+  "promotions.promotion.elligibility.cartProduct.verb.product.any": "es cualquiera entre",
+  "promotions.promotion.elligibility.cartProduct.verb.product.not.any": "no es cualquiera entre",
   "promotions.promotion.elligibility.cartProduct.verb.sku.any": "es cualquiera de estos SKUs",
   "promotions.promotion.elligibility.cartProduct.verb.sku.not.any": "no es cualquiera de estos SKUs",
   "promotions.promotion.elligibility.cartProduct.verb.itemPriceRange.gt": "con precio más alto que",
   "promotions.promotion.elligibility.cartProduct.verb.itemPriceRange.lt": "con precio inferior a",
   "promotions.promotion.elligibility.cartProduct.verb.itemPriceRange.between": "con precio entre",
   "promotions.promotion.elligibility.cartProduct.verb.fromToPrices": "con precio 'de'/'por'",
-  "promotions.promotion.elligibility.cartProduct.verb.seller.any": "es de uno de esos sellers",
+  "promotions.promotion.elligibility.cartProduct.verb.seller.any": "es de uno de esos vendedores",
```
Comment (ohmyguigs) [inline messages/es.json]: Actually `Sellers` is an internal term known in our platform, thats why we left this in english (should be upper-cased tho)
```suggestion
  "promotions.promotion.elligibility.cartProduct.verb.seller.any": "es de uno de esos Sellers",
```

File: messages/es.json [hunk 5]
```diff
@@ -138,19 +138,19 @@
   "promotions.promotion.elligibility.cartProduct.hasCategories.placeholder": "Seleccione las categorías deseadas…",
   "promotions.promotion.elligibility.cartProduct.hasProducts.placeholder": "Seleccione los productos deseados…",
   "promotions.promotion.elligibility.cartProduct.hasSkus.placeholder": "Seleccione los SKUs deseados…",
-  "promotions.promotion.elligibility.cartProduct.hasSellers.placeholder": "Seleccione los Sellers…",
+  "promotions.promotion.elligibility.cartProduct.hasSellers.placeholder": "Seleccione los vendedores…",
```
Comment (ohmyguigs) [inline messages/es.json]: Same goes here.
```suggestion
  "promotions.promotion.elligibility.cartProduct.hasSellers.placeholder": "Seleccione los Sellers…",
```

File: messages/es.json [hunk 6]
```diff
@@ -164,13 +164,13 @@
   "promotions.promotion.restriction.restrictTradePolicies.verb.not.any": "no es una de",
   "promotions.promotion.restriction.restrictTradePolicies.placeholder": "Seleccione una política comercial",
   "promotions.promotion.restriction.origin": "Origen",
-  "promotions.promotion.restriction.origin.marketplace": "Marketplace",
-  "promotions.promotion.restriction.origin.fulfillment": "Fulfillment",
+  "promotions.promotion.restriction.origin.marketplace": "Mercado",
```
Comment (ohmyguigs) [inline messages/es.json]: These two are also known terms
```suggestion
  "promotions.promotion.restriction.origin.marketplace": "Marketplace",
```

File: messages/es.json [hunk 7]
```diff
@@ -164,13 +164,13 @@
...
+  "promotions.promotion.restriction.origin.fulfillment": "Cumplimiento",
```
Comment (ohmyguigs) [inline messages/es.json]:
```suggestion
  "promotions.promotion.restriction.origin.fulfillment": "Fulfillment",
```

File: messages/es.json [hunk 8]
```diff
@@ -164,13 +164,13 @@
...
   "promotions.promotion.restriction.origin.marketplace.explanation": "Pedidos hechos directamente a su tienda",
-  "promotions.promotion.restriction.origin.fulfillment.explanation": "Pedidos donde tu tienda actúa como seller",
+  "promotions.promotion.restriction.origin.fulfillment.explanation": "Pedidos donde tu tienda actúa como vendedor",
```
Comment (ohmyguigs) [inline messages/es.json]: Here also
```suggestion
  "promotions.promotion.restriction.origin.fulfillment.explanation": "Pedidos donde tu tienda actúa como Seller",
```

File: messages/es.json [hunk 9]
```diff
@@ -190,15 +190,15 @@
...
+  "promotions.promotion.scopeSelector.collection.label": "Colección",
   "promotions.promotion.scopeSelector.collection.verb.any": "es una de",
   "promotions.promotion.scopeSelector.collection.verb.not.any": "no es una de",
   "promotions.promotion.scopeSelector.collection.placeholder": "Seleccione...",
   "promotions.promotion.scopeSelector.product.label": "Producto",
   "promotions.promotion.scopeSelector.product.verb.any": "es uno de",
   "promotions.promotion.scopeSelector.product.verb.not.any": "no es uno de",
   "promotions.promotion.scopeSelector.product.placeholder": "Seleccione...",
-  "promotions.promotion.scopeSelector.seller.label": "Seller",
+  "promotions.promotion.scopeSelector.seller.label": "Vendedor",
```
Comment (ohmyguigs) [inline messages/es.json]: Oops i missed this one
```suggestion
  "promotions.promotion.scopeSelector.seller.label": "Seller",
```

**Comment by Blask0** [comment]
> Nice! thanks so much @Blask0 🎉

Thanks for the opportunity :D , good luck ... if you need more help just let me know :p

**Comment by ohmyguigs** [comment]
Thanks @Blask0 You da real MvP!
Alt Text

**Pull request merged**